dev/com.ibm.ws.jca.cm/resources/com/ibm/ws/j2c/resources/J2CAMessages.nlsprops

```diff
@@ -112,7 +112,7 @@ NO_VALID_TRANSACTION_CONTEXT_J2CA0040.explanation=There was an internal error in
 NO_VALID_TRANSACTION_CONTEXT_J2CA0040.useraction=CMSG0002
 
 # --------------------------------
-POOL_MANAGER_EXCP_CCF2_0001_J2CA0045=J2CA0045E: Connection not available while invoking method {0} for resource {1}.
+POOL_MANAGER_EXCP_CCF2_0001_J2CA0045=J2CA0045E: Connection not available while invoking method {0} for resource {1}. Timed out waiting for {2} millisecond with {3} remaining waiting requests and current total connections used {4}.
```

Comment:
Should milliseconds be plural?  I suppose it could also be 1 millisecond. Or maybe millesecond(s)?  ID may know what we normally would do here.

Reply:
Updated adding the (s)